### DISCUSSION TITLE
How works the Username Changed feature?

### DISCUSSION THREAD

**Discussion by AdrianArmesto** [games647/FastLogin#731] [Q&A] [answered]
Hi, i'm interested on using this on my server, but before that, I want to know how exactly the plugin manages the username changed feature.

The Readme file says:
- Detect username changed and will update the existing database record.

Despite being sure that this plugin is doing its work I can't just put it without being a bit concerned about the behavior of this specific feature.

Imagine this situation:

- A player without Minecraft bought joins the server with the username JohnDoe
- A player with Minecraft joins the server with the name Alice
- Alice uses the /premium feature
- Days later Alice changes her username to JohnDoe

What would exactly happen with both users? 

Thanks!

**Comment by TuxCoding** [comment, marked as answer]
The plugin doesn't convert the user if there already an existing one.
